collect_sources.sh

```diff
@@ -28,6 +28,10 @@ mkdir -p $APTROOT/var/cache/apt/archives/partial
 	| (cd $APTROOT ; tar xf -)
 (cd / ; tar cf - /usr/lib/apt)	\
 	| (cd $APTROOT ; tar xf -)
+
+# no source package is provided by nodesource.list
+rm -f $APTROOT/etc/apt/sources.list.d/nodesource.list
```

Comment:
This is not what i meant. You are modifying the image. This is not ok. After the collec_sources script has been run the image must be the same as before.

Reply:
IMAGEDIR point to the image, APTROOT is just /tmp/apt.xxx on host

Reply:
Yes, you are right!